nixops/backends/libvirtd.py

```diff
@@ -163,14 +166,22 @@ def iface(n):
                 '    </interface>',
             ]).format(n)
 
+        def _make_os(defn):
+            return [
+                '<os>',
+                '    <type arch="x86_64">hvm</type>',
+                "    <kernel>%s</kernel>" % defn.kernel or '',
```

Comment:
or is redundant here if you only allow string type in Nix

Reply:
Right, fixed it in a separate commit to make review easier